src/js/authNear.js

```diff
@@ -55,7 +57,16 @@ async function login () {
 
   render()
 
-  if (window.ethInitialized) checkTransferStatuses(render)
+  if (window.ethInitialized) {
+    window.nearConnection.inFlightTransactions.drain(({ meta, tx }) => {
```

Comment:
I think I would do it with async iterators, so that you can work like this:

```
for await (let { meta, tx } of walletAccount.successfulTransactions) {
   // do something
}

for await (let { meta, tx } of walletAccount.failedTransactions) {
   // do something
}
```

Reply:
While I think many scenarios would be supported by these
`successfulTransactions` and `failedTransactions` iterators, my design
goals for this app (namely, don't export unnecessarily, keep related
logic together, and have the code work whether or not it generates a
redirect) lead me to propose an additional `completedTransactions`
option. We could certainly add all three to near-api-js.

Additionally, here I've added a `.find` method on completedTransactions.
Unlike `Array.prototype.find`, I suspect the `find` on these
`*Transactions` iterators would not merely return the found item, but
would remove it from the underlying data store, since an app would
presumably only want to deal with each of these once.

Let me know what you think, @vgrichina!